Just a build fix.

# missing mach_port_set_protected_payload and mach_port_clear_protected_payload RPCs

Index: hurd-debian/mach-defpager/Makefile
===================================================================
--- hurd-debian.orig/mach-defpager/Makefile
+++ hurd-debian/mach-defpager/Makefile
@@ -27,7 +27,7 @@ SRCS	:= default_pager.c kalloc.c wiring.
 OBJS 	:= $(SRCS:.c=.o) \
 	   $(addsuffix Server.o,\
 		       memory_object default_pager memory_object_default exc) \
-	   default_pager_replyUser.o
+	   default_pager_replyUser.o mach_portUser.o
 
 HURDLIBS:= ihash
 OTHERLIBS:= -lpthread
